src/wutil/dir_iter.rs

```diff
@@ -328,8 +328,12 @@ impl Iterator for Iter {
 
 #[cfg(test)]
 mod tests {
+    use std::fs::File;
+    use std::path::PathBuf;
+
+    use nix::sys::stat::Mode;
```

Comment:
We haven't put a lot of thought into import ordering yet;
for tests specifically I put everything in one block,
which lists the "super::*" dependencies first.
The idea is that this can make it slightly easier to see our test coverage.
Not sure.

I'll make this small edit before merging, so no need to update this.